contracts/BodhiToken.sol

```diff
@@ -9,13 +9,14 @@ contract BodhiToken is StandardToken, Ownable {
   string public constant symbol = "BOT";
   uint256 public constant decimals = 18;
 
+  // Token constants
+  uint256 public constant MAX_TOKENS_FOR_SALE = 60e6 ether; // 60 million BOT tokens for sale
+  uint256 public constant MAX_TOKEN_SUPPLY = 100e6 ether; // 100 million BOT tokens will ever be created
```

Comment:
I still suggest we use `60 * (10**6) * (10**decimals)` here. That reason is because it clearly tells us that the number of tokens is `60 * (10**6)` and the one token can be divided into `10**decimals`. Using `ether` is so confusing.

Reply:
Nice change adding the constants BTW, makes the code so much more clear!

Reply:
Agreed! ether was new to me, had to look it up. At least this is clear cut for the math.